### ISSUE TITLE
Stone generators not working in 1.19.2

### ISSUE THREAD

**Issue by philippewarren** [AstralOrdana/Immersive-Weathering#168]
**Describe the bug:**
Stone generators don't work.
Tried them all, none of them work except for Cobblestone and Basalt (which are both vanilla).

**Versions:**
Minecraft version: 1.19.2
Immersive Weathering version: 1.1.3
Moonlight version: 2.1.3
Forge version: 43.1.12, 43.1.47 (both)

**Other mods:**
No other mods

**Logs:**
[debug.log](https://github.com/AstralOrdana/Immersive-Weathering/files/9874622/debug.log)
[latest.log](https://github.com/AstralOrdana/Immersive-Weathering/files/9874623/latest.log)

**To Reproduce:**
1. Create a generator based on the [wiki](https://github.com/AstralOrdana/Immersive-Weathering/wiki/Custom-Generators)
2. Notice that the result is either normal cobblestone or nothing, based on the setup.

**Expected behavior:**
The specific stone type should be generated based on the stone type indicated in the wiki.

**Screenshots:**
![2022-10-26_20 17 54](https://user-images.githubusercontent.com/55722645/198162359-3d55302b-5bdf-4499-baff-60e4308827bb.png)
Shows both terracotta (flowing lava next to clay) and diorite (cobblegen with adjacent smooth quartz) failing to be converted.

**Comment by AstralOrdana** [comment]
that is not a valid generator setup

**Comment by AstralOrdana** [comment]
however, looking into this did uncover that the andesite generator is not working

**Comment by AstralOrdana** [comment]
fixed for future version, the data file was just missing for some reason

**Comment by philippewarren** [comment]
Hi, sorry, I think this should be reopened, as it doesn't address the bug I reported in the first place. I'm adding a new screenshot, closer to the one in the wiki, to better show that the generator I built is indead a valid setup.

![2022-10-29_12 28 58](https://user-images.githubusercontent.com/55722645/198842692-b3a0c73f-e666-4713-9724-02289fd2cb13.png)

Is it possible that the registration of the recipes from the datapack fails? I have confirmed that the datapack loads as I can get the "Wax On" advancement.

**Comment by philippewarren** [comment]
I also tried creating a custom datapack to test the customization system presented in the [wiki](https://github.com/AstralOrdana/Immersive-Weathering/wiki/Customization#fluid-generators).

Here is a screenshot of the directory tree. 
![image](https://user-images.githubusercontent.com/55722645/198844173-980ecbc2-44c8-4535-b4de-a2ee1b7739ab.png)

The pack.mcmeta format version is 10. I duplicated the content in two places, as I wasn't sure of the layout (in subdirectory `immersive_weathering` as is done in the mod, or directly in `data` as stated in the Acacia from cake tutorial).

The content of `truc.json` is this (from the tutorial, plus added target_self type):
```
{
    "adjacent_blocks": {
        "sides": [
            {
                "block_state": {
                    "Properties": {
                        "bites": "0"
                    },
                    "Name": "minecraft:cake"
                },
                "predicate_type": "minecraft:blockstate_match"
            }
        ],
        "up": {
            "block_state": {
                "Name": "minecraft:glass"
            },
            "predicate_type": "minecraft:blockstate_match"
        }
    },
    "fluid": "minecraft:lava",
    "generate": {
        "Name": "minecraft:acacia_log",
        "Properties": {
            "axis": "y"
        }
    },
    "type": "target_self"
}
``` 

This didn't help. The diorite generator from my comment above doesn't work either, and acacia isn't generated from this setup:
![2022-10-29_13 00 09](https://user-images.githubusercontent.com/55722645/198844215-bfcec5c7-a1b3-4f1d-9efe-8a25a962b821.png)

I hope this is helpful information.

**Comment by hunie2** [comment]
I also hope for this issue to be reopened. Even for the newest version 1.1.4 none of the generators seems to be working, even with no other mods installed.

**Comment by AstralOrdana** [comment]
there must be some issue on your end because I cannot replicate this in the latest version
![image](https://user-images.githubusercontent.com/62686557/200038844-2a746add-9abc-4398-a2fc-7662b1cd8245.png)
![image](https://user-images.githubusercontent.com/62686557/200038928-8b53ab95-6c7f-4448-b5c0-2c78a83e4d1b.png)

**Comment by hunie2** [comment]
Are these screenshots taken in fabric? It seems that the forge version may be the issue, as when I switched from forge to fabric, the generators had no problems working.
![2022-11-04_15 41 43](https://user-images.githubusercontent.com/117452847/200085882-8b77e23c-944c-446b-b517-1f4fde88ac84.png)
![2022-11-04_15 48 25](https://user-images.githubusercontent.com/117452847/200085940-5a31c079-11ab-4e9a-a270-6901532153b1.png)
![2022-11-04_15 47 12](https://user-images.githubusercontent.com/117452847/200085990-9834c812-52dc-4385-a39b-7a0670beb8e3.png)

**Comment by AstralOrdana** [comment]
confirmed not working on forge

**Comment by AstralOrdana** [comment]
fixed for 1.2.3
